proprietary/react-icons/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+<!-- @license CC0-1.0 -->
```

Comment:
I think this file is in the wrong location, it should be in `proprietary/assets`

Reply:
Moved it